client/src/components/User/DiskUsage/DiskUsageSummary.vue

```diff
@@ -70,6 +67,12 @@ export default {
             dismissCountDown: 0,
         };
     },
+    computed: {
+        ...mapGetters("user", ["currentUser"]),
+        quotaUsages() {
+            return [new QuotaUsage(this.currentUser)];
+        },
```

Comment:
I'm afraid this will break the WIP PR https://github.com/galaxyproject/galaxy/pull/14047/files#diff-0e951cc03e84227b59111640c05c69508fd7618df4fcc4a939a77dfdf70a1f60

The `api/users/current` request in `fetchQuotaUsage` was just a temporal placeholder to be replaced when the multiple quotas endpoint is ready. Maybe we can adapt the QuotaUsageProvider to just fetch from the store meanwhile?

Reply:
With the switch to pinia and composables that part would need to be redone anyway. Let's bring in the fix, I think the ready to merge PR should win over the WIP here.